custom/icds_reports/ucr/tests/test_birth_preparedness_form_ucr.py

```diff
@@ -25,6 +25,17 @@ def test_birth_preparedness_form(self):
                 "ifa_last_seven_days": None,
                 "eating_extra": 1,
                 "resting": 1,
+                "anc_weight": None,
+                "anc_blood_pressure": 0,
+                "bp_sys": None,
+                "bp_dia": None,
+                "anc_abnormalities": None,
+                "anc_hemoglobin": None,
+                "bleeding": None,
```

Comment:
Might be good to add a test for the `anc_details` stuff in the UCR to make sure it's being collected properly

Reply:
will do. I did test all of those in the expression debugger but it does seem like a good idea to formalize that